path_height_checker/PathHeightStatus[] path_statuses